dependent_type_theory.md

```diff
@@ -482,30 +482,33 @@ So `def` can also be used to simply name a value like this:
 def pi := 3.141592654
 ```
 
-`def` can take multiple input parameters.  Let's create one
+`def` can take multiple input parameters.  Let's define a function
 that adds two natural numbers:
 
 ```lean
-def add (x y : Nat) :=
```

Comment:
When the `add (x y : Nat)` example comes first, a newcomer might think that it's a function of two arguments (no currying). So it seems better to have `add (x Nat) (y : Nat)` come first, and then say that it can be abbreviated.